homeassistant/components/rointe/const.py

```diff
@@ -0,0 +1,34 @@
+"""Constants for the Rointe Heaters integration."""
+
+import logging
+
+LOGGER = logging.getLogger(__package__)
+
+DOMAIN = "rointe"
+DEVICE_DOMAIN = "climate"
+PLATFORMS: list[str] = ["climate", "sensor", "update"]
```

Comment:
Instead of strings, use `Platform`—example: https://github.com/home-assistant/core/blob/96c442349f10e7d9b4823d684c127a3ef5a41d0e/homeassistant/components/notion/__init__.py#L31